decred/decred/util/helpers.py

```diff
@@ -21,6 +22,25 @@
 from appdirs import AppDirs
 
 
+def sign(x):
+    """
+    x: number
+    """
+    if x < 0:
+        return -1
+    elif x > 0:
+        return 1
+    else:
+        return 0
+
+
+random.seed(0)
```

Comment:
Can't set the random seed in production code. I'm hesitant to add any functions here that are just for tests. How about a test utilities module somewhere?

Reply:
> Can't set the random seed in production code. I'm hesitant to add any functions here that are just for tests.

Makes sense.

> How about a test utilities module somewhere?

Test code does not live in packages and modules. Test discovery is performed by pytest according to its own rules. For cases like this, pytest offers [fixtures](https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions), I'll put `randBytes` in one.

The remaining question is whether setting the seed should be done in the fixture or in the test that uses it: I lean towards the latter.